Sources/Plasma/FeatureLib/pfPython/pyVaultPlayerInfoListNode.cpp

```diff
@@ -79,14 +73,13 @@ bool pyVaultPlayerInfoListNode::HasPlayer( uint32_t playerID )
     if (!fNode)
         return false;
 
-    hsRef<NetVaultNode> templateNode = new NetVaultNode;
-    templateNode->SetNodeType(plVault::kNodeType_PlayerInfo);
-    VaultPlayerInfoNode access(templateNode);
+    NetVaultNode templateNode;
+    templateNode.SetNodeType(plVault::kNodeType_PlayerInfo);
+    VaultPlayerInfoNode access(&templateNode);
     access.SetPlayerId(playerID);
-    
-    hsRef<RelVaultNode> rvn = fNode->GetChildNode(templateNode, 1);
-    
-    return (rvn != nil);
+
+    hsRef<RelVaultNode> rvn = fNode->GetChildNode(&templateNode, 1);
+    return rvn;
```

Comment:
This implicit cast to bool is misleading (it looks like you're returning the node)...